core/error.rs

```diff
@@ -372,6 +372,7 @@ impl ErrWithV8Handle {
   }
 }
 
+#[allow(clippy::non_send_fields_in_send_ty)]
```

Comment:
These are suspect. Probably ok?

Reply:
Yeah, It's probably going to be allowed by default https://github.com/rust-lang/rust-clippy/issues/8045